polyfills/WeakSet/config.json

```diff
@@ -3,12 +3,13 @@
 		"es6"
 	],
 	"browsers": {
-		"ie": "7 - *",
-		"ie_mob": "10 - *",
-		"safari": "*",
-		"chrome": "<38",
+		"ie": "7 - 11",
+		"ie_mob": "10 - 11",
+		"safari": "<9",
+		"chrome": "<36",
```

Comment:
chrome 36 doesn't support `new WeakSet(iterable)` - could we revert this back to chrome 38?